assessment/libs/stats.php

```diff
@@ -2196,4 +2197,393 @@ function dotplot($a,$label,$dotspace=1,$labelspace=null,$width=300,$height=150)
 	return showasciisvg($outst,$width,$height);
   }
 
+//---------------------------------------------ANOVA-Oneway F ratio-----------------------------------------
+// Function: anova1way_f(arr1,arr2, [arr3,...])
+// Returns F ratio and the corresponding P value as an array. 
+//
+// Parameters:
+// arr1, arr2, ...: Arrays in the form [2,3,4,5,...] 
+//  
+// Returns:
+// F ratio and the corresponding P value as an array in the form [F ratio, P value].
+
+function anova1way_f(... $arr){
+	$n=array();  
+	foreach($arr as $a){
+		if (!is_array($a)) { $a = explode(',',$a);};
+		$n[]=count($a);
+		if (count(array_unique($n))!=1) {
+			echo "Error: ANOVA requires the same length for all arrays";
+			return false;
+		}
+	}
+	if (count($n)<2) {
+		echo "Error: ANOVA requires at least two arrays";
+		return false;
+	}
+		
+	$numargs = func_num_args();
+	$args=func_get_args();

```

Comment:
I'm confused why you're using the spread operator `... $arr` in the function declaration, but then using `func_get_args` here, when this is returning the same thing that $arr would contain.